libs/src/sdk/sdkBrowserDist.ts

```diff
@@ -1,2 +1,12 @@
+import { ParseRequestError } from '../legacy'
+import { developmentConfig, productionConfig, stagingConfig } from './config'
 import { sdk } from './sdk'
+import { DiscoveryNodeSelector } from './services'
 ;(window as any).audiusSdk = sdk
+;(window as any).audiusSdk.DiscoveryNodeSelector = DiscoveryNodeSelector
+;(window as any).audiusSdk.config = {
+  staging: stagingConfig,
+  production: productionConfig,
+  development: developmentConfig
+}
+;(window as any).audiusSdk.ParseRequestError = ParseRequestError
```

Comment:
pardon my ignorance but why do we need to expose this?

Reply:
this is for if you use the browser dist, so like loading SDK via the CDN or something. You don't have the ability to import anything in that case, so I'm "exporting" by tagging along the global `audiusSdk` object. I think having the `ParseErrorRequest` exposed could be useful for consumers to be able to error check etc.